dask/delayed.py

```diff
@@ -23,6 +24,8 @@
 
 __all__ = ["Delayed", "delayed"]
 
+D = TypeVar("D")
```

Comment:
Nitpick: I usually see this type of variable assigned to the variable `T`

Reply:
Me too, though I saw that "T" was in graph_manipulation.py, which unless there is some use of globals, would not be an issue, yet "K" & "V" were used in utils.py.  I did not dig much deeper but I assume they are for Key and Value. I am happy to change to "T".  Thanks!